lib/typo_killer.ex

```diff
@@ -1,9 +1,23 @@
 defmodule TypoKiller do
+  @moduledoc """
+  Qu'est-ce que c'est
+
+  Fa-fa-fa-fa-fa-fa-fa-fa-fa
+
+  It's a funny app to search wrong typos inside projects and report
```

Comment:
```suggestion
  It's a funny app to search typos inside projects and report
```

Every typo is wrong 💀 😆

Reply:
HAHAHAHA, okay
wait a sec